firedrake/slate/slac/compiler.py

```diff
@@ -32,6 +32,7 @@
 from firedrake.petsc import get_petsc_variables
 from firedrake.utils import complex_mode, ScalarType_c, as_cstr
 from ufl.log import GREEN
+from ufl.domain import extract_unique_domain, extract_domains
```

Comment:
I think all of the changes in this file need to be reverted.

Reply:
reverted